twisted/words/protocols/irc.py

```diff
@@ -3604,7 +3604,7 @@ def assembleFormattedText(formatted):
 
     @rtype: C{str}
     @return: String containing mIRC control sequences that mimic those
-        specified by L{formatted}.
+        specified by C{formatted}.
```

Comment:
minor comment... maybe use I{formatted} as C{formatted} should be used for code example and this is in fact a reference to the formatted argument